homotopy-graphics/src/style.rs

```diff
@@ -22,3 +33,78 @@ impl Default for VertexShape {
         Self::Circle
     }
 }
+
+pub trait RenderableColor {
+    fn css(&self) -> String;
+    fn tikz(&self) -> String;
+}
+
+pub trait CssStylesheet {
+    fn css_stylesheet(&self, prefix: &str) -> String;
+    fn css_class(prefix: &str, generator: Generator, suffix: &str) -> String;
+}
+
+pub trait TikzStylesheet {
```

Comment:
Could you please move this into the tikz module?

Also it doesn't have to be a trait, it could just be a top-level function, but that's up to you.

Reply:
Great, will do...